src/jit/lower.cpp

```diff
@@ -1179,8 +1188,14 @@ void Lowering::LowerArg(GenTreeCall* call, GenTreePtr* ppArg)
         // Insert a copy to move float value to integer register.
         if ((call->IsVarargs() || comp->opts.compUseSoftFP) && varTypeIsFloating(type))
         {
-            var_types  intType = (type == TYP_DOUBLE) ? TYP_LONG : TYP_INT;
-            GenTreePtr intArg  = comp->gtNewOperNode(GT_COPY, intType, arg);
+            var_types intType    = (type == TYP_DOUBLE) ? TYP_LONG : TYP_INT;
+
+#ifdef _TARGET_ARM_
+            // We need gtOtherRegs[] for the case - copy from a double register to two int registers
+            GenTreePtr intArg = new (comp, GT_COPY) GenTreeCopyOrReload(GT_COPY, intType, arg);
+#else  // !_TARGET_ARM_
```

Comment:
This should not be `#ifdef _TARGET_ARM_`. It should always use `GenTreeCopyOrReload()`